changelog.md

```diff
@@ -11,6 +11,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Update plugin-core to version 0.14.0
 
+## [0.1.13] - 2023/03/16
```

Comment:
It's not a release pull request, put content in [Unreleased] section

Reply:
Done